scripts/tsconfig.json

```diff
@@ -13,40 +13,24 @@
     "esModuleInterop": true,
     "isolatedModules": true,
     "strictBindCallApply": true,
-    "lib": [
-      "dom",
-      "esnext"
-    ],
-    "types": [
-      "node",
-      "jest"
-    ],
+    "lib": ["dom", "esnext"],
+    "types": ["node", "jest"],
     "strict": false,
     "strictNullChecks": false,
     "forceConsistentCasingInFileNames": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
```

Comment:
Especially in Angular sandboxes sym-linking to internal Storybook modules doesn't work because of these two settings:

```json
"noImplicitOverride": true,
"noPropertyAccessFromIndexSignature": true,
```

Do you mind / do you think it makes sense to enable it per default in our project as well?

Reply:
```ts
"noImplicitOverride": true,
"noPropertyAccessFromIndexSignature": true,
```

I kind of like both, but I think we should first tackle the `strict` flags and especially `strictNullChecks`.